value.go

```diff
@@ -554,6 +554,8 @@ func (vlog *valueLog) init(db *DB) {
 	lf, err := InitDiscardStats(vlog.opt)
 	y.Check(err)
 	vlog.discardStats = lf
+	// See TestPersistLFDiscardStats for purpose of statement below.
+	db.logToSyncChan("End: vlog.init(db)")
```

Comment:
would it better to use enum here instead of strings?

Reply:
I like the idea still of logging strings, because it is actually kind of helpful to be able to have a string-channel for dumping log messages using the waitForMessage function.
I modified this to use string constants instead of just random strings however, which, I believe, will get much the same effect that you are looking for: we want to match on a very specific message.